### ISSUE TITLE
Gantt chart feature request: allow displaying only a portion of the Gantt chart

### ISSUE THREAD

**Issue by RohanTalip** [plantuml/plantuml#377]
At the moment, I can specify a project start to display the Gantt chart from that date, however tasks that I've indicated started earlier (and finished earlier) still show up:
```
@startuml
Project starts 2020-09-01
[A] starts 2020-08-01 and lasts 1 week
[B] starts 2020-09-01 and lasts 2 weeks
@enduml
```
[online](http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuGeeoizAJIvH00iKAnWP61de6bZg6XXoHJl6eefQ04KL4lDI578IYu62XWhbgQdPND5EiMYQKPKPWPKLSpcavgM0B0G0)

If I want to generate only a portion of my project, then I have to comment out or remove the tasks that are outside this window.

It would be great if I could specify a diagram something like this (it would only show task B) :
```
@startuml
Project displays 2020-09-01 to 2020-09-30
[A] starts 2020-08-01 and lasts 1 week
[B] starts 2020-09-01 and lasts 2 weeks
@enduml
```
[online](http://www.plantuml.com/plantuml/uml/SoWkIImgAStDuGeeoizAJIvHICaiBiX9h2nMC38mCj0riDGrC5GeoOVpZ0sueXrZ5S3QO6eiG6eIyr8KSXABWOA62kMfgTbSqKveoYnHbHc1bHLpEQJcfG0T0000)

**Comment by f0rdprefect** [comment]
Exactly what I am looking for right now. Good to know that I can stop searching now.

**Comment by The-Lum** [comment]
Hello @RohanTalip, @f0rdprefect, and all,

For that you can use `Print between` command as:
```puml
@startgantt

Print between 2020-09-01 and  2020-09-30

Project starts 2020-08-01
[A] starts 2020-08-01 and lasts 1 week
[B] starts 2020-09-01 and lasts 2 weeks
@endgantt
```
>[![](https://www.planttext.com/api/plantuml/svg/SoWkIImgIK_CAodXuWeeoimhKKXABIbFJSrJC38mCj0riDGrC5H8p4k1SuqDG0hpirAJInJ0cekXKXP0bLpHZh6Oec3zEOd5G4531Q3PsLpHJcZAB56L6O6L5NDnEQJcfO1T1m00)](https://www.planttext.com/?text=SoWkIImgIK_CAodXuWeeoimhKKXABIbFJSrJC38mCj0riDGrC5H8p4k1SuqDG0hpirAJInJ0cekXKXP0bLpHZh6Oec3zEOd5G4531Q3PsLpHJcZAB56L6O6L5NDnEQJcfO1T1m00)

_Ref._: [QA-11503](https://forum.plantuml.net/11503) 
_Duplicate_: #378

_Enjoy,_
Regards,
Th.

**Comment by swmatisa** [comment]
Nice, but there is a bug with printscale weekly:
`@startgantt

Print between 2020-09-01 and  2020-09-30
**printscale weekly**

Project starts 2020-08-01
[A] starts 2020-08-01 and lasts 1 week
[B] starts 2020-09-01 and lasts 2 weeks
@endgantt`

**Comment by The-Lum** [comment]
Hi all,

The last bug is now fixed on [v1.2024.0](https://github.com/plantuml/plantuml/releases/tag/v1.2024.0).

Thanks to PlantUML team.

Regards,
Th.
